### ISSUE TITLE
Homepage scrolls horizontally in IE11

### ISSUE THREAD

**Issue by mxsnyder** [baltimorecounty/baltimorecountymd.gov-assets#28]
The homepage scrolls horizontally in IE11. I've narrowed it down to the search box being the issue, and the snippet below shows the change I made in the inspector that seemed to make the problem go away. 

```CSS
.carousel-wrapper .search-container {
    position: absolute;
    right: 50%; /* changed from left: -50% */
    z-index: 999;
    -moz-transform: translateX(-50%);
    -o-transform: translateX(-50%);
    -webkit-transform: translateX(-50%);
    transform: translateX(50%);
}
```

**Comment by martypowell** [comment]
@danfox01 very strange, but the current homepage css is not consistent with our master branch, which is causing the issue. @mxsnyder actually fixed this in this [commit](https://github.com/baltimorecounty/baltimorecountymd.gov-assets/commit/1845ae59a83b1bac54f4d9c848160c6efac45d40#diff-a118afc8c727f1d68ec0368156566064). 

Development has been updated with the current master version of the homepage styles and does not show this bug. Let me know if you want to run some tests on the homepage before we schedule this for release.

**Comment by martypowell** [comment]
Fixed in #31
